File: app/src/main/java/com/example/resourcesapp/ui/gitRepos/GitReposViewModel.java
package com.example.resourcesapp.ui.gitRepos;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class GitReposViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public GitReposViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is gallery fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/com/example/resourcesapp/ui/gitRepos/GitReposViewModel.kt
package com.example.resourcesapp.ui.gitRepos

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class GitReposViewModel : ViewModel() {
    private val mText: MutableLiveData<String>

    init {
        mText = MutableLiveData()
        mText.value = "This is gallery fragment"
    }

    val text: LiveData<String>
        get() = mText
}